mmfunctions

```diff
@@ -148,6 +155,10 @@ _premis_event_list(){
     if [ "$eventDetail" = "makewaveform" ] ; then
         eventType="creation"
     fi
+    
+    if [ "$eventDetail" = "makefingerprint" ] ; then
+        eventType="creation"
```

Comment:
haha yeah- the fixity vocab is also a spot where I cheated and went with a local vocabulary :-P